src/components/v2/Icon/icons/index.tsx

```diff
@@ -34,6 +34,7 @@ export { ReactComponent as checkInline } from './checkInline.svg';
 export { ReactComponent as mark } from './mark.svg';
 export { ReactComponent as arrowShaft } from './arrowShaft.svg';
 export { ReactComponent as notice } from './notice.svg';
+export { ReactComponent as warning } from './warning.svg';
```

Comment:
I think we currently have this SVG because it is being rendered with the name `attention`

Reply:
removed warning icon, used attention.